model/host/host.go

```diff
@@ -1987,7 +1987,7 @@ func GetContainersOnParents(d distro.Distro) ([]ContainersOnParents, error) {
 func getNumNewParentsAndHostsToSpawn(pool *evergreen.ContainerPool, newHostsNeeded int, ignoreMaxHosts bool) (int, int, error) {
```

Comment:
I know you didn't write this, but while you're here maybe "newhostsNeeded" should be changed to newContainersNeeded because this was mildly confusing to read.

Reply:
Apparently each container is itself on a "host", but the hosts are in a sense virtual, and are usually colocated on a physical EC2 instance. This confuses me as well.